builds/misc/mqtt-perf.yaml

```diff
@@ -0,0 +1,53 @@
+# MQTT Performance tests pipeline
+# Each platform separated into stages
+# Each stage consists of 
+# - setup job for test host (pull and start 'mqttd' container). See templates/mqtt-perf-setup.yaml
+# - setup job for publisher node and subscriber node (mqtt-benchmark tool writen in go). See templates/mqtt-perf-setup.yaml
+# - a set of test cases. See templates/mqtt-perf-test-case.yaml.
+
+# Pipeline requires custom agent pool with at least two physical agents:
+# - with 'mqtt-perf-tests-pub' demand - for publisher node
+# - with 'mqtt-perf-tests-sub' demand - for subscriber node
+# And a SSH service connection `mqttd-host-ssh` for the test host machine 
+# to manage and run 'mqttd' docker container there.
+
+trigger: none
+pr: none
+
+# github repo for our fork of mqtt-benchmark tool.
+resources:
+  repositories:
+  - repository: mqttbenchmark
+    type: github
+    endpoint: github.com_mqtt-benchmark
+    name: vadim-kovalyov/mqtt-benchmark
+
+variables:
+  benchmarkToolPath: $(Agent.WorkFolder)/mqtt-benchmark
+
+stages:
+################################################################################	
+- stage: linux_amd64
+################################################################################	
+  variables:
+    brokerImageName: $(broker.image.amd64)
+    brokerAddress: $(broker.address.amd64)
+    clientDop: 4 # the number of max threads that can execute mqtt-benchmark tool code simultaneously
+
+  pool: Default
+  jobs:
+    - template: templates/mqtt-perf-setup.yaml
+    - template: templates/mqtt-perf-run.yaml
+
+################################################################################	
+- stage: eclipse_mosquitto
```

Comment:
this is temp, just to compare with mosquitto for now. We will remove it later (or keep if we find it valuable to compare).